test/models/tradingpremises/TradingPremisesSpec.scala

```diff
@@ -372,4 +372,16 @@ class TradingPremisesSpec extends AmlsSpec {
     }
   }
 
+  "TradingPremises.filter" must {
+    "filter out any TradingPremises() instances which have the 'Deleted' status" in {
+
+      val completeModelDeletedStatus = completeModel.copy(status = Some(StatusConstants.Deleted), hasChanged = true, hasAccepted = true)
+      val tradingPremisesSeq: Seq[TradingPremises] = Seq(completeModel, completeModelDeletedStatus, completeModel)
```

Comment:
Can you add `TradingPremises()` to this list of models? We need to check that blank/default instances of that case class are filtered in this function